run_local.sh

```diff
@@ -1,3 +1,29 @@
 #!/bin/bash
+
 export DOCKER_BUILDKIT=1
-docker compose up --build
\ No newline at end of file
+export ENVIRONMENT=${1:-development}
+export BACKEND_BASE_URL="http://host.docker.internal:8010"
+
+git clone https://github.com/ministryofjustice/cla_backend ../cla_backend
```

Comment:
Can this be moved to /tmp or perhaps create a tmp folder in the project which has it's content ignored in this repo